android-auto-app/src/main/AndroidManifest.xml

```diff
@@ -8,7 +8,6 @@
     <uses-permission android:name="android.permission.ACCESS_FINE_LOCATION" />
 
     <uses-permission android:name="androidx.car.app.NAVIGATION_TEMPLATES" />
-    <uses-permission android:name="androidx.car.app.ACCESS_SURFACE" />
```

Comment:
isn't this permission required any more?

Reply:
Move this inside the SDK since it is required to use the android-auto extension. Developers would inherit the permission

Reply:
Similar conversation here https://github.com/mapbox/mapbox-maps-android/pull/1004#discussion_r787695698

In the other thread we're considering moving the other permission to the sdk as well
`<uses-permission android:name="androidx.car.app.NAVIGATION_TEMPLATES" />`